agent/tool-scripts/perf

```diff
@@ -191,13 +191,14 @@ case "$mode" in
 			popd > /dev/null 2>&1
 			# Generate the initial report.
 			$tool_bin $report_opts -i ${tool_output_dir}/perf.data --stdio > ${tool_output_dir}/perf-report.txt 2> ${tool_output_dir}/perf-report.err
-			xz --threads=0 ${tool_output_dir}/perf.data
 			# only postprocess the perf report if the default options are used
 			if echo $record_opts | grep -E -q -- -g\|--callgraph; then
-				debug_log "$script_name: not going to run perf postprocess script since this report contains a callgraph"
+				# create flamegraph if --callgraph option is used
+				$script_path/postprocess/$script_name-postprocess ${tool_output_dir} callgraph
```

Comment:
To avoid repeating ourselves (DRY principle), can we re-write as:

```
callgraph_opt=""
if echo $record_opts | grep -E -q -- -g\|--callgraph; then
    callgraph_opt="callgraph"
fi
$script_path/postprocess/$script_name-postprocess ${tool_output_dir}  ${callgraph_opt}
```
```